src/components/books.css

```diff
@@ -1,14 +1,139 @@
-.book {
-  border: 1px black solid;
+.books {
```

Comment:
- [ ] You are almost there, but the styling is still looking fairly different from the template, please use the provided font, add some space between the cards and align all the elements similar to the template:

|Your design|Template|
|---|---|
|<img width="1425" alt="Screen Shot 2022-07-16 at 9 29 31 PM" src="https://user-images.githubusercontent.com/83717285/179383469-ade2a126-fad7-41fa-8e86-29cd210d0651.png">|<img width="1440" alt="Screen Shot 2022-07-16 at 10 57 09 PM" src="https://user-images.githubusercontent.com/83717285/179383457-698e91b6-c327-48df-b2d6-b914c93d0ed0.png">|